sklearn/decomposition/dict_learning.py

```diff
@@ -142,7 +146,7 @@ def _sparse_encode(X, dictionary, gram, cov=None, algorithm='lasso_lars',
 # XXX : could be moved to the linear_model module
 def sparse_encode(X, dictionary, gram=None, cov=None, algorithm='lasso_lars',
                   n_nonzero_coefs=None, alpha=None, copy_cov=True, init=None,
-                  max_iter=1000, n_jobs=1):
+                  max_iter=1000, n_jobs=1, pool=None):
```

Comment:
I think that pool is a misleading name, it should rather be named `parallel` (or `managed_parallel_context` to be explicit).